Commands/BowerInstallCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Pingpong\Modules\Commands;
+
+use Illuminate\Console\Command;
+use Pingpong\Modules\Module;
+use Pingpong\Modules\Repository;
+
+class BowerInstallCommand extends Command
+{
+
+    /**
+     * @var string
+     */
+    protected $signature = 'module:bower-install {--production}';
+
+    /**
+     * @var string
+     */
+    protected $description = 'Runs `bower install` on all modules, where applicable';
+
+    /**
+     * @var Repository;
+     */
+    protected $modules;
+
+    /**
+     * BowerInstallCommand constructor.
+     * @param Repository $module
+     */
+    public function __construct(Repository $modules)
+    {
+        parent::__construct();
+
+        $this->modules = $modules;
+    }
+
+
+    public function fire()
+    {
+        // Check bower is installed
```

Comment:
It already reports when Bower is not installed, or when something went wrong during the check itself, in `isBowerInstalled()`.